packages/utils/src/getOptionMatchingSimpleDiscriminator.ts

```diff
@@ -0,0 +1,36 @@
+import get from 'lodash/get';
+import { PROPERTIES_KEY } from './constants';
+import { RJSFSchema, StrictRJSFSchema } from './types';
+
+export default function getOptionMatchingSimpleDiscriminator<T = any, S extends StrictRJSFSchema = RJSFSchema>(
```

Comment:
Please add the same documentation you put into the docs above here.

Reply:
done